far/farlang.templ.m4

```diff
@@ -2881,7 +2881,7 @@ upd:"ClearType-friendly redraw (can be slow)"
 "Limpiar rediseño (puede ser lento)"
 "Použiť &ClearType (pomalšie)"
 "Modalità ClearT&ype (può rallentare)"
-"Перемальовка з підтримкою ClearType (може бути повільною)"
+"Друк з підтримкою ClearType (може бути повільною)"
```

Comment:
While technically correct, "друк" is predominantly associated with paper printing.

Maybe "вивід", as in "output", or just "rendering", since it is already used twice above?

Reply:
yeah, it hard to realize the correct term in Ukrainian for that..
Replaced with "Рендерінг" for this case.